switch-to-classicpress-from-wordpress.php

```diff
@@ -0,0 +1,106 @@
+<?php 
+/*
+Plugin Name:       Switch to ClassicPress from WordPress
+Plugin URI:        https://github.com/classicpress/classicpress-migration-plugin/
+Description:       Switch to ClassicPress from WordPress and say Goodbye to Gutenberg.
```

Comment:
Please remove "and say Goodbye to Gutenberg"

Reply:
Ok to remove "and say Goodbye to Gutenberg"